src/UCTSearch.cpp

```diff
@@ -239,21 +287,11 @@ int UCTSearch::get_best_move(passflag_t passflag) {
         }
     }
 
-    int visits = m_root.get_visits();
-
     // if we aren't passing, should we consider resigning?
     if (bestmove != FastBoard::PASS) {
-        // resigning allowed
-        if ((passflag & UCTSearch::NORESIGN) == 0) {
-            size_t movetresh = (m_rootstate.board.get_boardsize()
-                                * m_rootstate.board.get_boardsize()) / 4;
-            // bad score and visited enough
-            if (bestscore < ((float)cfg_resignpct / 100.0f)
-                && visits > 500
-                && m_rootstate.m_movenum > movetresh) {
-                myprintf("Score looks bad. Resigning.\n");
-                bestmove = FastBoard::RESIGN;
-            }
+        if (should_resign(passflag)) {
+            myprintf("Eval (%.3f) looks bad. Resigning.\n", bestscore);
```

Comment:
Scores are winrates, so convert to percentage and add a %%.

Reply:
I don't agree that it's a winrate (in the same way a p value != probability in statistics) but will fix